aip/general/0162.md

```diff
@@ -258,13 +218,13 @@ when to commit a new revision, such as:
 - Commit a new revision when something important happens
 - Commit a new revision when the user specifically asks
 
-APIs **may** use any of these strategies. APIs that want to commit a revision
-on user request **should** handle this with a `Commit` custom method:
+APIs **may** use any of these strategies. APIs that want to commit a revision on user request should
+handle this with a custom method on the resource:
 
 ```proto
-rpc CommitBook(CommitBookRequest) returns (Book) {
+rpc CommitBookRevision(CommitBookRevisionRequest) returns (BookRevision) {
```

Comment:
should a tag field be in the request as well? it might make it easier to tag a specific revision as you're creating it.